tools/bumper/cnaoRepoCommands.go

```diff
@@ -22,7 +22,8 @@ type gitCnaoRepo struct {
 }
 
 const (
-	repoUrl = "https://github.com/kubevirt/cluster-network-addons-operator"
+	repoUrl         = "https://github.com/kubevirt/cluster-network-addons-operator"
+	allowListString = "components.yaml,data/*,test/releases/99.0.0.go,pkg/components/components.go"
```

Comment:
If we convert this into a string slice we don't need `getAllowedList`

Reply:
the idea was to keep the repo-specific stuff (like allowListString content and the repo url) in one place. I didn't want to hide it in a functin, and since string slice cannot be a const, I preferred using a string and converting in `getAllowedList` to string slice.